### ISSUE TITLE
Add `@babel/template`-like package

### ISSUE THREAD

**Issue by fengzilong** [benjamn/ast-types#369]
Write the builders syntax by hand is really annoying, can we provide some package like @babel/template or add an online playground to parse the code to builders syntax

**Comment by rajasegar-c** [comment]
@fengzilong Actually I am working on one playground to build AST nodes through code for jscodeshift and Glimmer templates here. 
https://rajasegar.github.io/ast-builder/
Please let me know this is what you are looking for. Appreciate any PRs or issues.
And thanks for the `@babel/template` pointer btw.
For this project, let's hear from @benjamn

**Comment by fengzilong** [comment]
Yeah, that's the playground I need, but there are some issues with yours

try this snippet

```js
module.exports = {
  components: {
    'demo': demo
  }
}
```

the 'demo' becomes `j.identifier('undefined')`

and it will be much better if we can format the code with prettier

thanks for reply @rajasegar-c

**Comment by rajasegar** [comment]
@fengzilong Thanks for trying it out and reporting the issues. It's a bug  I need to fix and yes formatting needs some work.

**Comment by rajasegar** [comment]
Can you give me an example for the ES6 export you were trying, have you tried changing the parser to babel or babylon?
Fixed the module.exports and formatting

**Comment by fengzilong** [comment]
@rajasegar sorry for my late reply, I just see the notification from github. I tried and ES6 export has been fixed, and formatting works perfectly👍
